eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -21,3 +21,181 @@ function Get-NamepspacesFromDll($dllPath) {
     }
     return $namespaces | Sort-Object -Unique
 }
+
+function DownloadNugetPackage($package, $version, $destination) {
+    # $PackageSourceOverride is a global variable provided in
+    # Update-DocsMsToc.ps1. Its value can set a "customSource" property.
+    # If it is empty then the property is not overridden.
+    $customPackageSource = Get-Variable -Name 'PackageSourceOverride' -ValueOnly -ErrorAction 'Ignore'
+    if ($customPackageSource) {
+        # Download package from devop public feeds
+        Write-Host "Download from public feeds: $customPackageSource"
+        # Invoke the download link and store it into destination
+        nuget install -FallbackSource $customPackageSource $package -Version $version -DirectDownload -DependencyVersion Ignore -OutputDirectory $destination
```

Comment:
Go ahead with `$package.$version` as folder name since we might need to run both preview and latest. Make the script robust.